week-6/Homework/mandatory/2-quotegenerator/style.css

```diff
@@ -1 +1,72 @@
 /** Write your CSS in here **/
+body {
+  background: -webkit-linear-gradient(rgb(12, 68, 255), rgb(13, 219, 219));
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  width: 100%;
+  height: 100%;
+}
+.container {
+  display: block;
+  align-items: right;
+  margin-top: 10%;
+  height: 50%;
+  width: 55%;
+}
+h1 {
+  font-size: 32 px;
+  width:100%;
+  color: white;
+  justify-content: center;
+  hyphens:auto; 
+  /* text-shadow: 1px 2px 25px rgb(255, 255, 255);
+
+  background-clip: text;
+  -webkit-text-fill-color: transparent; */
+  text-align: top;
+  float: left;
```

Comment:
You're using a number of contradictory alignment rules here - `align-items: right;` in the container, `justify-content: center;` and `float: left;` on the h1... 

Generally avoid `float` rules completely. Stick to using flexbox for most alignment, and if you want to align text inside a box, use `text-align: left/right/center`